_posts/2021-07-19-changelog-151.markdown

```diff
@@ -0,0 +1,140 @@
+
+    <img src="/assets/posts/changelog-151/parameter-info.png" alt="Parameter info" width="700px">
+
+* `.match` [postfix template](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-generation.html#postfix-completion) improvements:
+
+  * [#7476] Always place caret into the first match arm
+
+  * [#7346] Insert `.as_str()` automatically when matching `String` (by [@Kobzol])
+
+  * [#7328] Allow filling pats after generating match arms (by [@Kobzol])
+
+* [#7425] Enable crates local index (`org.rust.crates.local.index` [experimental feature]) on the nightly plugin channel by default. The new index is aimed to improve completion and code insight related to crate dependencies, mainly affecting `Cargo.toml` files.
+  If you face any issues, please feel free to provide us with feedback via `Help | Create New Issue` action right from your IDE
+
+* [#4109] Provide LLDB pretty-printers to render `OsStr`, `CString`, `CStr`, `PathBuf`, `Path` std types in IDE debugger
+
+    <img src="/assets/posts/changelog-151/lldb-pretty-printers.png" alt="LLDB pretty-printers" width="700px">
+
+* [#7148] Add `New crate version available` inspection for `Cargo.toml` files. This inspection warns about possibly outdated dependency versions.
+  Note, it requires `org.rust.crates.local.index` [experimental feature] (by [@Kobzol])
+
+* [#7457] Add completion for items inside visibility restrictions (`pub(<...>)`) (by [@Kobzol])
+
+* [#7345] Support type aliases in `Substitute associated type` intention (by [@Kobzol])
+
+* [#7456] Annotate trailing commas in TOML inline tables as errors
+
+## Performance Improvements
+
+* [#7437] Optimize creation of crates local index (by [@Kobzol])
+
+## Fixes
+
+* [#7471] Suggest traits in completion only if the path must resolve to a trait (`impl <...> for type` or `T: <...>`) (by [@Kobzol])
+
+* [#7462] Fix parsing of expressions and statements containing block subexpressions (by [@afetisov])
+
+* [#7451] Only complete ancestor mods in visibility restriction paths (`pub(in <...>)`) (by [@Kobzol])
+
+* [#7436] Fix `Create struct` intention behaviour when invoked from inside an `impl` block (by [@Kobzol])
+
+* [#7415] Fix false-positive error on a multi-line array inside TOML inline table
+
+* [#7392] Fix [`Move Statement Up/Down`](https://www.jetbrains.com/help/idea/working-with-source-code.html#move-statements) action (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>Up</kbd>/<kbd>Down</kbd>) on block expressions inside a comma-separated list (by [@Kobzol])
+
+* [#7473] Fix `Add impl block` intention for structures with const generic parameters (by [@afetisov])
+
+* [#7474] Fix `Create lifetime parameters from usage` for items with const generic parameters (by [@afetisov])
+
+* [#6788] Correctly fill empty function call arguments with `Fill missing arguments` intention (by [@Kobzol])
+
+* [#7403] Fix false-positive `Cannot resolve directory` warnings on paths with wildcards (`*` or `?`) in TOML files (by [@Kobzol])
```

Comment:
I'd add a note that it's about paths in workspace section only where glob patterns are available